README.md

```diff
@@ -16,17 +17,22 @@ Netty is an asynchronous event-driven network application framework for rapid de
 
 For the detailed information about building and developing Netty, please visit [the developer guide](https://netty.io/wiki/developer-guide.html).  This page only gives very basic information.
 
-You require the following to build Netty:
+You will require the following to build Netty:
 
-* Latest stable [OpenJDK 8](https://adoptium.net/)
+* Latest stable [OpenJDK 11](https://adoptium.net/) for Netty 5, [OpenJDK 8](https://adoptium.net/) for older releases.
```

Comment:
I was tempted to suggest changing adoptium link to a vendor neutral OpenJDK-finder like `https://foojay.io/download/` but will let someone else make that decision.

Reply:
Let's keep it like this for now.